playbooks/common/openshift-master/scaleup.yml

```diff
@@ -56,6 +56,8 @@
   - openshift_facts
   - openshift_docker
 
+- include: ../openshift-loadbalancer/config.yml
+
 - include: ../openshift-master/config.yml
```

Comment:
I assume the load balancer health checks will prevent a bad host from being put into rotation, but should we update the load balancer after the master has been deployed?